iothub/device/tests/Pipeline/RetryDelegatingHandlerTests.cs

```diff
@@ -129,7 +129,7 @@ public async Task RetryOneMessageHasBeenTouchedTransientExceptionOccuredSuccess(
 
             var contextMock = Substitute.For<PipelineContext>();
             contextMock.ConnectionStatusChangesHandler = new ConnectionStatusChangesHandler(delegate (ConnectionStatus status, ConnectionStatusChangeReason reason) { });
-            var innerHandlerMock = Substitute.For<IDelegatingHandler>();
+            using var innerHandlerMock = Substitute.For<IDelegatingHandler>();
```

Comment:
Why would this require a `using` statement?

Reply:
I had been running into the "ambiguous arguments" exception that we've seen on NSubstitute, so I made this change in an effort to clean up all involved resources in a test.
Do you think there'd be a problem in marking the mocked handlers to be disposed at the end of each test?